packages/wp-story-editor/src/api/media.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@web-stories-wp/design-system';
+import { flattenFormData } from '@web-stories-wp/story-editor';
```

Comment:
Just move this to `wp-story-editor`?

Reply:
Absolutely!! 
Moved in https://github.com/google/web-stories-wp/pull/8966/commits/d6d13ce7fcde9312809e3519592d67c2fa0b1f4c